commit 5f71060f11e463c2446241a60fdd8ad297edcaff (refs/changes/36/2536/2)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2017-09-11T14:28:40+00:00 (2 years, 1 month ago)
    
    OS-6339 lint error due to xsaveopt merge
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/uts/intel/ia32/ml/float.s b/usr/src/uts/intel/ia32/ml/float.s
index 0d33365764..f154a96851 100644
--- a/usr/src/uts/intel/ia32/ml/float.s
+++ b/usr/src/uts/intel/ia32/ml/float.s
@@ -277,10 +277,6 @@ void
 xsave_ctxt(void *arg)
 {}
 
-void
-xsaveopt_ctxt(void *arg)
-{}
-
 /*ARGSUSED*/
 void
 xsaveopt_ctxt(void *arg)
